dashboard-for-desktop/articles/dashboard-designer/appearance-customization/conditional-formatting.md

```diff
@@ -32,63 +21,80 @@ Comparison rules used in conditional formatting can be divided into the followin
 * [Bar Color Ranges](conditional-formatting/bar-color-ranges.md) - Allows you to visualize numeric values using bars whose colors are contained in the specified color set.
```

Comment:
```suggestion
* [Bar Color Ranges](conditional-formatting/bar-color-ranges.md) - Visualizes numeric values with bars. Bar colors are contained in a specified color set.
```

Reply:
Please change to:
* [Bar Color Ranges](conditional-formatting/bar-color-ranges.md) - Visualize numeric values as bars. Values within a range display a specific color.

(Sorry, I just noticed that "Visualizes" should be "Visualize")